src/string_editor_window.cpp

```diff
@@ -17,6 +17,11 @@ static bool is_linebreak( const uint32_t uc )
     return uc == '\n';
 }
 
+static bool break_before( const uint32_t uc )
+{
+    return uc >= 0x2E80;
```

Comment:
CJK characters are contained in several ranges in Unicode, so is this one inequality check enough?

As a reference this is the CJK character check logic in Cocos2D:
https://github.com/cocos2d/cocos2d-x/blob/90f6542cf7fb081335f04e474b880d7ce8c445a1/cocos/base/ccUTF8.cpp#L149-L160

Reply:
I see. I'll update the comparison and the one in `foldstring` which I copied from (which seems to ultimately come from [a 2013 commit](https://github.com/CleverRaven/Cataclysm-DDA/commit/f23a08660363cfeacfeba32027a9cf3131a2cdc3#diff-1054ee29cb29bafa93a575b91d8cba6bf0ff9ea0c3904ee37cc8bf4aea1452ebR883-R886)), probaly in a separate PR.

Reply:
Thanks!